lib/theme.js

```diff
@@ -1,8 +1,22 @@
+const palette = {
+  white: '#FFFFFF',
+  blue: '#3C76f0',
+  concrete: '#F2F2F2',
+  boulder: '#777777'
+}
+
 export const theme = {
+  dimens: {
```

Comment:
Is this a typo?

Reply:
`dimensions` is too long 😄  but I can use it.